Jenkinsfile

```diff
@@ -90,7 +90,24 @@ pipeline {
                             failure_artifacts: env.STAGE_NAME
                     }
                     // runTest handles SCM notification via stepResult
-                } // stage('grep JUnit results tests error case')
+                } // stage('grep JUnit results tests error case 1')
+                stage('grep JUnit results tests error case 2') {
+                    agent {
+                        docker {
+                            image 'rockylinux:8'
+                            label 'docker_runner'
+                        }
+                    }
+                    steps {
+                        runTest script: '''set -ex
+                                           rm -f *.xml
+                                           echo "<errorDetails bla bla bla/>" > \
```

Comment:
Do you have a link to an example of this new format?  I'm just curious what's all changed.

Reply:
New to us, but not to the Junit plugin, it is in their test data.

https://github.com/jenkinsci/junit-plugin/blob/master/src/test/resources/hudson/tasks/junit/junitResult.xml
https://github.com/jenkinsci/junit-plugin/blob/master/src/test/resources/hudson/tasks/junit/junit-report-6700.xml

Still have not found a definitive spec for JUnit.   The definitive code for it is supposed to be in Apache Ant source modules.

Reply:
Do you have a link from our tests though?  It's difficult to meaningfully review the changes here without the data on hand that the changes are being made to accomodate.

Reply:
No links are available, which made this a bit harder to diagnose.
I had to look at the internal Jenkins job storage to find the junit XML file.